eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelTypes.scala

```diff
@@ -223,8 +223,10 @@ object ChannelFlags {
   val Empty = 0x00.toByte
 }
 
-sealed trait ChannelVersion
+case class ChannelVersion(bits: BitVector) {
+  require(bits.size == 4 * 8, "channel version takes 4 bytes")
```

Comment:
Size requirement is hardcoded in 3 different places, which not define it in `ChannelVersion`'s companion object ?